src/test/java/net/datafaker/script/ProviderGenerator.java

```diff
@@ -77,14 +84,14 @@ private void createCreator(File file, String key, Set<String> strings, ProviderT
 
         System.out.println("package " + providerType.getPackageName() + ";");
         System.out.println();
-        System.out.println("import net.datafaker.providers.base.AbstractProvider;");
+        System.out.println("import " + AbstractProvider.class.getName() + ";");
         System.out.println();
         System.out.println("/**");
-        System.out.println(" * @since 1.8.0");
+        System.out.println(" * @since 2.0.2");
```

Comment:
I guess the behaviour is slightly different but the functionality was still introduced in 1.8

Reply:
Or is this something that's tweaked (or supposed to be) on every use?

Reply:
this is a generator of a boiler plate for new provider, so in theory it should contain actual version since it is invoke for actual version.
May be in future it will be better to use something like maven -properties plugin to put version from there or something like that to make it bumping automatically here